Tiltfile

```diff
@@ -22,6 +22,8 @@ watch_file('result')
 if os.path.exists('result'):
    k8s_yaml('result/crds.yaml')
 
+k8s_kind('Deployment', image_json_path='{.spec.template.metadata.annotations.image}')
```

Comment:
Need to move this into templating...

Reply:
Done in https://github.com/stackabletech/operator-templating/pull/294

Reply:
Reverted in https://github.com/stackabletech/operator-templating/pull/297

But we are probably bringing it back as per slack huddle.